### ISSUE TITLE
[Bug] Flashy artifacts on water when used with Oculus

### ISSUE THREAD

**Issue by DedMaxim** [Hantonik/FancyBlockParticles#3]
### Issue description:

If FBP is installed together with Oculus, water randomly changes opacity. This behavior can be seen in a video I recorded. If only one of these 2 mods is installed, the issue does not happen. At least, that's what I observed. Re-logging into the world/server fixes it for that specific spot but later on issue appears on other places. Due to its random nature, this issue is quite annoying visually.

---Environment---

Computer:
Intel Xeon E5-2680v2
Colorful iGame Nvidia GTX 1070
4x8Gb DDR3 ECC REG Samsung RAM
Windows 10 22H2, Atlas mod
Nvidia driver 537.58

Mods:
Minecraft 1.20.1
Forge 47.2.20
FBP 1.1.1.0-beta
Oculus 1.6.15a

The modpack is assembled through CurseForge app

https://youtu.be/6XKbkBaYZVM

### Steps to reproduce:

The issue is a bit tricky to reproduce, since, as I said, it appears randomly. It appears in some spots at some specific angles and height above/below water. I discovered this bug randomly, while observing the world generation in my modpack. So to find a spot with such behavior, I would recommend installing these 2 mods, going into spectator and flying above bodies of water, constantly changing camera angles and height above the water

### FBP version:

1.1.1.0-beta

### Minecraft version:

1.20.4

### Modloader:

Forge

### Modloader version:

47.2.20

### The latest.log file:

[latest.log](https://github.com/Hantonik/FancyBlockParticles/files/14459691/latest.log)

**Comment by Hantonik** [comment]
Hi, thanks for reporting!
However, I'm having difficulty reproducing the issue. Could you please provide me with your Minecraft, Oculus, and FBP settings?
This information would help in resolving the problem more effectively.

**Comment by DedMaxim** [comment]
Whole config folder and options.txt. Will this do?

[minecraft.zip](https://github.com/Hantonik/FancyBlockParticles/files/14487249/minecraft.zip)

**Comment by DedMaxim** [comment]
Note: This folder includes lots of other configs but for this test I disabled all the mods that are not listed. So everything said above is still valid

**Comment by DedMaxim** [comment]
Updated my video driver to 551.61 - made no difference

**Comment by mosharky** [comment]
I'm actually having this issue too, but disabling Nvidium in video settings (I'm running it through Connector) fixes this for me, despite still having Oculus

**Comment by DedMaxim** [comment]
> I'm actually having this issue too, but disabling Nvidium in video settings (I'm running it through Connector) fixes this for me, despite still having Oculus

Hm. Well, I'm not using Nvidium because my graphics card doesn't support it. Since you're using it, means you own RTX 3000 or 4000. It could be the architectural difference between our cards that causes the reproduction of this bug being this different

**Comment by DedMaxim** [comment]
Another thing I noticed is if I start the game with FBP disabled, no visual artifacts will happen. However, if I turn it on while in the world, then this bug appears. And if I turn it off now, this bug won't go away until the game is restarted

**Comment by mosharky** [comment]
> Since you're using it, means you own RTX 3000 or 4000.

Yeah, I own a 3090. It could be related to some weird graphics thing

**Comment by DedMaxim** [comment]
Checked with newer version of the mod, 1.1.1.1-beta, the bug is still present

**Comment by DedMaxim** [comment]
Tried launching Fabric version on Forge using Sinytra Connector to check if the bug is present in such configuration but the game just crashes on launch

**Comment by Hantonik** [comment]
I'm sorry, but I can't reproduce the bug even after testing the mod with Oculus and the configs you provided. 
I'm afraid I can't help.

**Comment by DedMaxim** [comment]
As I got a notification about this issue, I went to test things again, to see if something magically changed and I found out why you couldn't reproduce the issue.
Right now I'm working on a modpack and so I'm building a map for a server. Because of it, I have to fly around a lot, do lots of things in multiple places. And as I went to check this bug again, I just realized that it only happens in Spectator mode!

https://drive.google.com/file/d/1S9U_Wf2P2LyvbQp6RVPhfo9j6gIvYpc5/view?usp=sharing

Which is good by itself, meaning actual players can use the mod just fine and won't be seeing any funky stuff. No real need to fix this issue, in such case

**Comment by Hantonik** [comment]
Actually, I conducted tests in both Creative and Spectator modes, and as I mentioned before, there were no effects whatsoever.
However, I'm glad to hear that the bug won't cause any inconvenience during regular gameplay.
